pages/linux/semanage.md

```diff
@@ -0,0 +1,24 @@
+# semanage
+
+> SELinux Policy Management tool.
+> More information: <https://linux.die.net/man/8/semanage>.
+
+- Output local customizations:
+
+`semanage -S {{store}} -o {{output_file}}`
+
+- Input local customizations:
```

Comment:
What precisely do you mean by this?

Reply:
Take a set of commands from a specified file and load them in a single transaction. May be a better way of putting it

Reply:
Ah, I see. Thanks for the clarification!

`Import` would work too, but your phrase is fine :-)